Enhancement/5692 vulnerability scanner documentation (#29506)
wazuh/wazuh: docs/ref/modules/vulnerability-scanner/flatbuffers.md

# Flatbuffer schemas

The Vulnerability Detector makes use of the FlatBuffers tool. FlatBuffers is a library that enables high-performance data serialization and deserialization without the need of unpacking or parsing, providing direct access to the required information.

> Although the synchronization events received by Remoted are in JSON format, they require to augmentate the event data with additional **agent context** within this module. As a result, deserializing and re-serializing the data becomes unavoidable. 

Reviewer: ```suggestion
Although the synchronization events received by Remoted are in JSON format, the event data needs to be augmented with additional **agent context** within this module. As a result, deserializing and re-serializing the data becomes unavoidable. 
```
